pipeline/electron/build-publish-electron-flavors.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+steps:
+    - script: yarn build:electron:all
+      displayName: build all electron flavors
+
+    - task: CopyFiles@2
+      displayName: 'copy files to publish'
+      inputs:
+          SourceFolder: '$(System.DefaultWorkingDirectory)'
+          Contents: |
+              **/drop/**
+              electron-builder.yml
+              **/extension/icons/**
+              **/src/electron/resources/**
+              **/pipeline/scripts/**
+          TargetFolder: '$(System.DefaultWorkingDirectory)/builtproduct'
```

Comment:
can this be a variable? so we re-use it between copy files target folder and publish path to publish?

Reply:
discussed offline, will include in future PR